### DISCUSSION TITLE
Differentials and AWD greyed out

### DISCUSSION THREAD

**Discussion by atledreier** [Mogli12/KeyboardSteer#740] [Q&A]
Hi!

I've been using this mod for a good while now, and I LOVE it!

I just recently started playing on a dedicated server, and since then my AWD and diff settings have been greyed out, even in single player.
Any idea what could cause that to happen?

**Comment by Mogli12** [comment]
This happens if VCA cannot identify the differentials or if the vehicle has tracks.

**Comment by atledreier** [comment]
It happens for all vehicles now. I've seen it on tracked vehicles, and it makes sense.

But I'm not seeing the setting in the vehicle menu, even if my friend on the same server sees it. So it's definitely something on my end that changed somehow.

**Comment by Mogli12** [comment]
I have no idea. Maybe you try to post the log.txt of client and server?

**Comment by atledreier** [comment]
[log.txt](https://github.com/Mogli12/VehicleControlAddon/files/6422681/log.txt)

Here's my log. This happens in single player too, so it's not server/MP specific.

I did move my install folder, and mod folder to another drive recently, I don't know if that may be related? 
It is reading and saving my settings in VCA, so it seems to know where the files go, but you never know, might be related somehow.

**Comment by Mogli12** [comment]
Ah, ok. If you use TSX_EnhancedVehicle, then VCA does not control the differentials.

**Comment by atledreier** [comment]
I've been playing with VCA and EV in unison for a good while now, and I've been controlling the differentials through VCA, both buttons and HUD. I disabled diffs in EV to make that work, and it has.

I'm completely stumped as to why it doesn't any more, though.

I completely wiped all my settings for all mods and keybinds to see if it made a difference, but it didnt.

I've got the diffs back in single player, but on my dedi it just refuse to work. My friends play happily with the exact same mods on the same server, and it works for them....  I'm just completely stumped at this point.

**Comment by Mogli12** [comment]
I use EVs function to check if it operates the differentials. It is lines 887-888 in vehicleControlAddon.lua.
